core/views.py

```diff
@@ -14,6 +17,19 @@ def get_page_size(self, request):
             return self.max_page_size
         return super().get_page_size(request)
 
+# Permit to send feedback emails from users
+@api_view(['POST'])
+def feedback(request, format=None):
+    data = request.data
+    if len(data['message']) < 2:
```

Comment:
```suggestion
    if len(data.get("message", "")) < 2:
```
If there is no message in the POST data, `get` won't throw an error.